src/_P053_PMSx003.ino

```diff
@@ -183,7 +189,7 @@ boolean Plugin_053_process_data(struct EventStruct *event) {
     addLog(LOG_LEVEL_DEBUG, log);
   }
 
-  if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
+  if (loglevelActiveFor(LOG_LEVEL_DEBUG) && PCONFIG(0) != PMS3003_TYPE) { // Count values not available on PMS2003 and PMS3003
```

Comment:
Judging the comment, this check doesn't seem complete.

Reply:
Well, the 'count' values are not available on the mentioned models (and the constant covers both models). Probably Uncrustify would add a couple of round braces, but it should work out fine.

Reply:
More like the comment suggests 2 types, where you only test for one.

Reply:
Tried to improve the comment and added those round braces.